src/MagnumPlugins/BasisImporter/Test/README.md

```diff
@@ -0,0 +1,65 @@
+Creating input PNG files
```

Comment:
This is still a copypaste leftover from the PngImporter ... and I think the `toolchains` update as well? :)